Cargo.toml

```diff
@@ -57,13 +57,13 @@ members = [
 ]
 exclude = [
   "fuzz",
-  "oak_enclave_shim",
+  "oak_enclave_binary",
   "oak_functions/loader/fuzz",
-  "oak_functions_enclave",
-  "oak_tensorflow_enclave",
+  "oak_functions_app",
```

Comment:
If we go this way we should rethink how we name our Linux variants. Alternatively we can indicate here that these target the Restricted Kernel (perhaps we should invent a short, snappy name) rather than Linux.